### PR TITLE
Windows link: 2.4.1 -> 2.4.4

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Comment by marioba** [comment]
Is it not possible to have a link pointing directly to the latest release?

**Comment by m-kuhn** [comment]
> Is it not possible to have a link pointing directly to the latest release?

I have researched if it's possible directly from github and didn't find anything, (at least as long as we have a version number in the filename).
The most promising approach I see at the moment is to add a cronjob that updates it.

**Comment by nirvn** [comment]
@marioba , @m-kuhn , let me try something.

**Comment by nirvn** [comment]
@marioba , @m-kuhn , added linux while I was at it. @mbernasocchi reported it working quite well, and that's my experience too.

File: documentation/get-started/index.en.md [hunk 1]
```diff
@@ -34,7 +34,13 @@ tx_slug: documentation_get-started_index
 === ":material-microsoft-windows: Windows"
 
     Windows support is currently in beta status. [Download the latest
-    builds](https://github.com/opengisch/QField/releases/download/v2.4.1/qfield-v2.4.1-windows-x64.exe)
+    builds](https://github.com/opengisch/QField/releases/download/v2.4.4/qfield-v2.4.4-windows-x64.exe)
```
Comment (nirvn) [inline documentation/get-started/index.en.md:37]:
```suggestion
    builds](https://qfield.org/get_latest/?platform=windows)
```

File: documentation/get-started/index.en.md [hunk 2]
```diff
@@ -34,7 +34,13 @@ tx_slug: documentation_get-started_index
...
+    and leave us a note how it works.
+
+=== ":material-linux: Linux"
+
+    Linux support is currently in beta status. [Download the latest
+    builds](https://github.com/opengisch/QField/releases/download/v2.4.4/qfield-v2.4.4-linux-x64.AppImage)
```
Comment (nirvn) [inline documentation/get-started/index.en.md:43]:
```suggestion
    builds](https://qfield.org/get_latest/?platform=linux)
```

**Review by suricactus — APPROVED** [review]

**Pull request merged**